code/script.js

```diff
@@ -15,37 +20,190 @@ const showMessage = (message, sender) => {
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/usercoffee.png" alt="User" />  
       </section>
     `
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/botcoffee.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hello there. I'm Cuppa, your virtual coffee assistant. What might be your name?", 'bot');
+}
+
+// User tells their name to the bot
+const userNameInput = event => {
+  event.preventDefault();
+  userName = nameInput.value;
+  nameInput.value = "";
+  showMessage(`I'm ${userName}.`, 'user');
+  setTimeout(drinkTypeOptions, 1500);
```

Comment:
`setTimeout(drinkTypeOptions, 1500);` Great choice for the time. It's not too short, and not too long so the customer have to wait or start to wonder if there's anything happening.